src/Microsoft.AspNet.Mvc.Core/ActionResults/ViewExecutor.cs

```diff
@@ -70,7 +72,14 @@ public static class ViewExecutor
             {
                 try
                 {
-                    var viewContext = new ViewContext(actionContext, view, viewData, tempData, writer);
+                    var viewContext = new ViewContext(
+                        actionContext,
+                        view,
+                        viewData,
+                        tempData,
+                        writer,
+                        htmlHelperOptions);
```

Comment:
We should be really careful about adding anything **options** to a context. These contexts are intended to be constructable for unit tests.